packages/thirdweb/src/extensions/prebuilts/deploy-marketplace.test.ts

```diff
@@ -0,0 +1,20 @@
+import { describe, expect, it } from "vitest";
+import { ANVIL_CHAIN } from "~test/chains.js";
+import { TEST_CLIENT } from "~test/test-clients.js";
+import { TEST_ACCOUNT_A } from "~test/test-wallets.js";
+import { isAddress } from "../../utils/address.js";
+import { deployMarketplaceContract } from "./deploy-marketplace.js";
+
+describe.runIf(process.env.TW_SECRET_KEY)("deploy marketplace test", () => {
+  it("should deploy marketplace contract", async () => {
+    const address = await deployMarketplaceContract({
```

Comment:
dont we already do this in the other marketplace tests?

Reply:
yeah I'm closing this. I was looking at the code coverage via Codecov. We were actually skipping all the marketplace tests because they are slow and flaky (or flaky due to being slow)